src/v2/components/CopyableCorrelationId.tsx

```diff
@@ -0,0 +1,43 @@
+import { Check, Copy, X } from "@phosphor-icons/react"
+import { useState } from "react"
+import { useCopyToClipboard } from "usehooks-ts"
+
+export const CopyableCorrelationId = ({
+  correlationId,
+}: { correlationId: string }) => {
+  const [, copyToClipboard] = useCopyToClipboard()
+  const [isCopied, setIsCopied] = useState(false)
+  const [hasFailed, setHasFailed] = useState(false)
+
+  const handleCopy = () => {
```

Comment:
We have a custom `useCopyToClipboard` hook in `/src/v2/hooks`, let's use that here